.github/workflows/warnings-tests.yml

```diff
@@ -0,0 +1,21 @@
+name: Run Warnings Tests
+
+on: [push]
+
+jobs:
+  Tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.12"]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install
+      run: |
+        pip install -e .[dev]
+    - name: Test with pytest
+      run: pytest -Werror --cov=./ --cov-report=xml
```

Comment:
I thought a space was needed between `-W` and `error`?

Reply:
Nope!